src/main/resources/plugin.yml

```
@@ -24,7 +24,7 @@ description: Customize Minecraft's mechanics with simple scripts written in plai
 
 authors: [Njol, Mirreski, 'SkriptLang Team', Contributors]
 
-website: https://skriptlang.github.io/Skript
+website: https://www.skriptlang.org
```

```suggestion
website: https://skriptlang.org
```
i don't think it's necessary to specify the www subdomain

---

It's not, indeed.

---

Let's get this change added so we can merge @EquipableMC

---

Alright 😄